docs/_docs/windows.md

```diff
@@ -81,6 +90,7 @@ the site generation process. It can be done with the following command:
 ```sh
 $ chcp 65001
 ```
+#### Time-Zone Management (Chocolatey method)
```

Comment:
@ashmaroli I moved it outside of the Chocolatey installation directions, so it has it's own section. I just can't decide if it makes more sense at the very end of the page, or the beginning, or what. I just thought since each tutorial had it's own mention of Time-Zone management and auto regeneration, that would be the make the most sense organizationally.